src/lang/TranslationsRU.js

```diff
@@ -0,0 +1,142 @@
+                'Вы вынуждены выйти из игры. Перенаправляем на домашнюю страницу через 5 секунд ...',
+            waitForOtherPlayersToFinish:
+                'Ожидайте, пока другие игроки закончат игру ...',
+        },
+        Maps: {
+            makeGuess: 'Предположить догадку',
+            guess: 'Угадать',
+            nextRound: 'Следущий раунд',
+            viewSummary: 'Посмтреть Сводку',
+            exit: 'Выход',
+            reset: 'Рестарт',
+            playAgain: 'Играть заново',
+            infoWindow: {
+                Distance: 'Дистанция',
+                Points: 'Очки',
+            },
+        },
+        Header: {
+            language: 'Язык',
+            about: 'Про Проект',
+            limitation: 'Лимиты',
+            contact: 'Контакты',
+        },
+        HeaderGame: {
+            room: 'КОМНАТА',
+            round: 'РАУНД',
+            distance: 'ДИСТАНЦИЯ',
+            kmaway: '{value} КМ отсюда',
+            score: 'ОЧКИ',
+        },
+        next: 'ДАЛЕЕ',
+        cancel: 'ОТМЕНА',
+        DialogCustomMap: {
+            title: 'Пользовательская карта',
+            invalid: 'Неверный GeoJSON',
+            text: 'Текст',
+            url: 'Ссылка',
+            file: 'Файл',
+            edit: 'Редактировать карту',
+            fileLabel: 'Выбрать GeoJSON файл',
+        },
+        History: {
+            title: 'История',
+            date: 'Дата',
+            mode: 'Режим',
+            time: 'Время',
+            distance: 'Дистанция',
+            points: 'Очки',
+            rank: 'Ранк',
+            search: 'Поиск',
+        },
+
+        DialogRoom: {
+            singlePlayer: 'Одиночная игра',
+            withFriends: 'Игра с друзьями',
+            invalidRoomName: 'Неверное название. Попробуйте другое.',
+            inProgress:
+                'Прямо сейчас первый игрок создает комнату. Подождите и попробуйте еще раз.',
+            roomIsFull:
+                'Эта комната уже заполнена. Пожалуйста, попробуйте другую.',
+        },
+
+        CardRoomName: {
+            title: 'Введите название комнаты.',
+        },
+        CardRoomPlayerName: {
+            title: 'Введите имя игрока.',
+        },
+        CardRoomSize: {
+            title: 'Установите размер комнаты.',
+        },
+        CardRoomTime: {
+            title: 'Установите ограничение по времени.',
+            infinite: 'Бесконечность',
+        },
+        CardRoomDifficulty: {
+            title: 'Установите уровень сложности.',
+            easy: 'Легкий (мир)',
+            medium: 'Средний (страна)',
+            hard: 'Сложный (город)',
+        },
+        DialogSummary: {
+            summaryMsgSingle:
+                'Вы попали <strong>{distance}</strong> km отсюда! Your score is <strong>{points}</strong> points!',
+            summaryMsgMulti:
+                '<strong>{playerName}</strong> количество очков <strong>{points}</strong> ! (<strong>{distance}</strong> kmaway)',
```

Comment:
`kmaway` last ;)

Reply:
https://pastebin.com/NdYTL1g4 edited one. GitHub don't wanna upload the file.